File: LLM-Translations/GPT-4/codenet/Java/C/s832608243.c
#include <stdio.h>
#include <math.h>
#include <stdbool.h>

int main() {
    int a, b;
    scanf("%d %d", &a, &b);
    bool flag = false;
    
    for (int i = 1; i <= 100000; i++) {
        if (floor(1. * i * .08) == a && b == floor(.1 * i)) {
            printf("%d\n", i);
            flag = true;
            break;
        }
    }
    
    if (!flag) {
        printf("-1\n");
    }
    
    return 0;
}

// 

File: LLM-Translations/GPT-4/codenet/Java/Python/s832608243.py
import math

a, b = map(int, input().split())
flag = False
for i in range(1, 100001):
    if math.floor(1. * i * .08) == a and b == math.floor(.1 * i):
        print(i)
        flag = True
        break

if not flag:
    print(-1)
    

